Add filemanager plugin and update hugo picture

hacdias opened the pull request:
That ^

mholt commented:
Nice, thanks!! It would probably be important to mention in the top of the docs that the user should cover it with some kind of authentication, even `basicauth`.

hacdias commented:
Oh sure! :D

mholt commented on public/docs/filemanager.html:
```diff
@@ -0,0 +1,69 @@
+			</p>
+			
+			<p>
+				It is extremely important for security reasons to cover the path of filemanager with some kind of authentication. You can use, for exemple, <a href="../basicauth"><code>basicauth</code></a> directive.
```
"for **example**"

hacdias replied:
I'm joining portuguese with english! Awesome! ahahah. Thanks  :D

mholt commented:
Thanks Henrique! It'd be really cool if there was a screenshot. :smiley: Especially with your updated UI that you showed me in Slack the other day.

Do you want to add one? Be sure to run it through tinypng.com first. ;)

hacdias commented:
Hello!!

Yup, I'll add a picture, but only after fishing this. I have to review the hugo docs too. :)

hacdias commented:
Hey @mholt 

I've changed the hugo pic and added a new one to filemanager. Yes, I passed them through tinypng.com :D See them and tell me if you like it.

mholt commented:
Beautiful, thanks!

hacdias commented:
Eheh you're welcome!

MSF-Jarvis commented:
The site hasn't been built yet, I can't download another fantastic plugin from @hacdias after having been a fan `hugo`, this has been a shame

hacdias commented:
Hey @MSF-Jarvis! I'm glad you liked ```hugo``` plugin. Thanks! The new version of ```caddyserver.com``` website will only be released after announcing (officially) the final build of Caddy 0.9. Anyway, both ```filemanager``` and ```hugo``` are not totally finished yet.

MSF-Jarvis commented:
I hate the waiting game :sob:
